### PR TITLE
[main]:EOS:14666:Modularize L10N JSON File Preboarding

### PR DESCRIPTION
**Pull request by JayshreeAmore**
# UI

 CSM UI : Modularize L10N JSON File Preboarding

## Problem Statement
<pre>
  <code>
    Story Ref (if any):EOS-14666
  </code>
</pre>
## Unit testing on RPM done
<pre>
  <code>
  No
  </code>
</pre>
## Problem Description
<pre>
  <code>
  </code>
</pre>
## Solution/Screenshots

 
image
image
image
image

<pre>
  <code>
  </code>
</pre>
## Unit Test Cases
<pre>
  <code>
   1)tested  welcome page
   2)tested login page
   3)tested agreement page

  </code>
</pre>

### CODE REVIEW

File: gui/src/components/preboarding/license-agreement.vue [hunk 1]
```diff
@@ -81,7 +81,7 @@
                 class="cortx-btn-tertiary"
                 @click="cancelAgreement"
               >
-                {{ $t("common.cancel") }}
+                {{ $t("admin.cancel") }}
```
Comment (giftedUI) [inline gui/src/components/preboarding/license-agreement.vue:84]: We can leave out common related translations, please revert all common related changes.

File: gui/src/components/preboarding/preboarding.json [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            "getStated": "Click on Start to proceed with the onboarding steps.",
+            "login-label": "Login",
+            "login-btn": "Login",
+            "user-name-placeholder": "Username",
+            "password-placeholder": "Password",
+            "user-name-required": "Username is required.",
+            "password-required": "Password is required.",
+            "login-failed": "Login failed !",
+            "login": "Login"
+
```
Comment (giftedUI) [inline gui/src/components/preboarding/preboarding.json:36]: remove additional new line spaces

**Review by giftedUI — CHANGES_REQUESTED** [review]

File: gui/src/components/preboarding/license-agreement.vue (hunk 1 above)
Comment (JayshreeAmore) [reply gui/src/components/preboarding/license-agreement.vue]: done

File: gui/src/components/preboarding/preboarding.json (hunk 2 above)
Comment (JayshreeAmore) [reply gui/src/components/preboarding/preboarding.json]: done

**Review by giftedUI — APPROVED** [review]

**Review by namrata-khake-ui — APPROVED** [review]

**Review by kupranay — APPROVED** [review]

**Pull request merged**